### PR TITLE
Adding GetEWSFolder

### PR DESCRIPTION
**Pull request by lperryp**
## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Adding a new automation for getting emails from multiple folders at inbox at oncee

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [x] Code Review

### CODE REVIEW

File: Scripts/GetEWSFolder/GetEWSFolder.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from CommonServerPython import *
+
+
+def convert_mail_to_json(item, folder):
```
Comment (erezh31) [inline Scripts/GetEWSFolder/GetEWSFolder.py]: It's better to split here the JSON for subject, body, html so we will have all the information. We might expand it in the future and use more information

File: Scripts/GetEWSFolder/GetEWSFolder.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+def main():
+    folders_paths_str = demisto.args()['foldersPaths']
+    folders_paths_list = [folder.strip() for folder in folders_paths_str.split(',')]
+    path_to_mails = {folder: [] for folder in folders_paths_list}  # type: Dict[str, List[str]]
+    for folder in folders_paths_list:
+
+        res = demisto.executeCommand('ews-get-items-from-folder', {
+            'folder-path': folder,
```
Comment (erezh31) [inline Scripts/GetEWSFolder/GetEWSFolder.py]: try to add the "isPublic" argument here

File: Scripts/GetEWSFolder/GetEWSFolder.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    isArray: false
+    name: isPublic
+    predefined:
+      - 'true'
+      - 'false'
+    required: false
+    secret: false
+  - default: false
+    defaultValue: '1000'
+    description: Limit number of items.
```
Comment (erezh31) [inline Scripts/GetEWSFolder/GetEWSFolder.yml]: Number of items per folder

File: Playbooks/playbook-Get_Mails_By_Folder_Pathes.yml [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+      dedupThreshold: {}
+      hashSeed: {}
+      input:
+        simple: ${lastCompletedTaskEntries}
+      inputType: {}
+      outputFormat: {}
+      preProcessType: {}
+      removeShortTextThreshold: {}
+      textFields:
+        simple: subject-body
```
Comment (erezh31) [inline Playbooks/playbook-Get_Mails_By_Folder_Pathes.yml]: you will need to change it to "subject,body|html" after you will change the JSON

**Comment by erezh31** [comment]
@anara123 need to handle docs

File: Playbooks/playbook-Get_Mails_By_Folder_Pathes.yml [hunk 5]
```diff
@@ -0,0 +1,194 @@
+id: d178ab0b-2368-474b-826a-9fdb716eaac8
```
Comment (anara123) [inline Playbooks/playbook-Get_Mails_By_Folder_Pathes.yml]: the id and the name must be the same

**Comment by anara123** [comment]
@kirbles19 please review

**Comment by yaron-libman** [comment]
Completed the tech docs review.

**Pull request merged**